packages/components/inputs/creatable-select-input/src/creatable-select-input.tsx

```diff
@@ -391,6 +395,7 @@ const CreatableSelectInput = (props: TCreatableSelectInputProps) => {
           maxMenuHeight={props.maxMenuHeight}
           menuPortalTarget={props.menuPortalTarget}
           menuShouldBlockScroll={props.menuShouldBlockScroll}
+          closeMenuOnSelect={props.closeMenuOnSelect}
```

Comment:
I think it would be great to also add the default value to the `defaultProps` attribute of the component.